docs/source/contents/4_draft_mapping.rst

```diff
@@ -2152,6 +2152,622 @@ Drop all values in mods:part.
+
+relatedItem[@type='host'][@displayLabel='Bibliographic Citation']
+-----------------------------------------------------------------
+Use Case
+^^^^^^^^
+This XPath only appears 1264 times in the Arrowmont Collection, specifically the Arrow of Pi Beta Phi sub-collection.
+
+Justification
+^^^^^^^^^^^^^
+XPath
+^^^^^
+:code:`relatedItem[@type='host'][@displayLabel='Bibliographic Citation']`
+
+Decision
+^^^^^^^^
+The `dcterms:bibliographicCitation` predicate was selected for these values.
+
+`Example record - arrow:1 <https://digital.lib.utk.edu/collections/islandora/object/arrow:1/datastream/MODS/view>`_
+
+.. code-block:: xml
+
+    <relatedItem type="host" displayLabel="Project">
+      <titleInfo>
+        <title>From Pi Beta Phi to Arrowmont</title>
+      </titleInfo>
+    </relatedItem>
+    <relatedItem displayLabel="Project Part" type="host">
+      <titleInfo>
+        <title>The Arrow of Pi Beta Phi</title>
+      </titleInfo>
+    </relatedItem>
+    <relatedItem displayLabel="Bibliographic Citation" type="host">
+      <titleInfo>
+        <title>The Arrow, Volume 27, Number 1</title>
+      </titleInfo>
+    </relatedItem>
+
+.. code-block:: turtle
+
+    @prefix dcterms: <http://purl.org/dc/terms/> .
+
+    <https://example.org/objects/1> dcterms:bibliographicCitation "The Arrow, Volume 27, Number 1" .
+
+relatedItem[@type="otherVersion"]
+---------------------------------
+Use Case
+^^^^^^^^
+`relatedItem[@type="otherVersion"]` is used to indicate identifying information about another version of the resource. It appears in the Van Vactor and Arrowmont metadata; it is used, respectively, to identify an alternate version of the sheet music or the scrapbook that holds the image.
+
+Justification
+^^^^^^^^^^^^^
+
+XPath
+^^^^^
+:code:`relatedItem[@type='otherVersion']/identifier` OR
+:code:`relatedItem[@type='otherVersion']/location/url`
+
+Decision
+^^^^^^^^
+See the following section on `relatedItem/identifier[@type]`.
+
+I'm not sure about the relevance of `relatedItem[@type='otherVersion']/location/url`. In theory, it would look something like this, but it would need to be added to our metadata/RDF after migration:
+
+`Example record - arrpgimg:319 <https://digital.lib.utk.edu/collections/islandora/object/arrpgimg:319/datastream/MODS/view>`_
+
+.. code-block:: xml
+
+    <relatedItem type="otherVersion">
+      <titleInfo>
+         <title>Gaitlinburg from All Sides</title>
+      </titleInfo>
+      <location>
+         <url>https://digital.lib.utk.edu/collections/islandora/object/arrowmont%3A16</url>
+      </location>
+    </relatedItem>
+
+.. code-block:: turtle
+
+    @prefix dcterms: <http://purl.org/dc/terms/> .
+
+    <https://example.org/objects/1> dcterms:hasVersion <uri-of-source-scrapbook-in-new-system> .
+
+relatedItem/identifier[@type]
```

Comment:
I think @type='local' can be grouped into the category "relatedItem[@type='series'][@displayLabel='Project']" since these are treated the same, correct? This might simplify the Use case description below (there are multiple use cases given here).

Reply:
@mlhale7 - I'm not sure. Since we're not migrating the `[@type='series'][@displayLabel='Project']` xpath, it doesn't seem to make sense to include a separate xpath in that section. 

I'll try splitting these into three groups.